memcached.c

```diff
@@ -7984,6 +7989,8 @@ static void usage(void) {
            "   - ssl_ca_cert:         PEM format file of acceptable client CA's\n"
            "   - ssl_wbuf_size:       size in kilobytes of per-connection SSL output buffer\n"
            "                          (default: %u)\n", settings.ssl_wbuf_size / (1 << 10));
+    printf("   - ssl_session_cache:   enable server-side SSL session cache, to support session\n"
```

Comment:
@dormando do we have to prefix this flag with `enable`?

Reply:
I don't think so actually. Double checking the help output, convention is to just name the feature. Disabling it gets a `no_` prefix though.